### PR TITLE
Initial hubhub wiring

### PR DESCRIPTION
**Pull request by lukewestby**
Starting to connect the UI to the data

https://deploy-preview-8--smvmt-microsite-dev.netlify.app/

Screen Shot 2020-05-18 at 10 26 31 PM

### CODE REVIEW

File: gatsby-site/gatsby-node.js [hunk 1]
```diff
@@ -42,8 +42,8 @@ const { createRemoteFileNode } = require("gatsby-source-filesystem")
  * @property {string | null} twitter
  * @property {string | null} signup_link
```
Comment (faarwa) [inline gatsby-site/gatsby-node.js:44]: do we need to check for undefined for these fields too?

File: gatsby-site/gatsby-node.js [hunk 2]
```diff
@@ -175,3 +175,11 @@ const fileNodeFromRemoteFile = async (helpers, remoteFile) => {
...
   return fileNode.id
 }
+
+/**
+ * @param {string} input
+ * @returns {string}
+ */
+const parseName = input => {
+  return input.replace(/sunrise(\s+movement)?/i, "").trim()
+}
```
Comment (faarwa) [inline gatsby-site/gatsby-node.js:185]: Is this because some hub names can already contain Sunrise?

File: gatsby-site/src/pages/index.js [hunk 3]
```diff
@@ -16,32 +16,33 @@ const MainContent = styled.div`
...
+        hubLogo={data.hub.logo.childImageSharp}
+        hubName={data.hub.name}
+        hubWebsite={data.hub.website}
       />
-    </MainContent>
-  </Layout>
-)
+      <MainContent>
+        <About
+          hubName="PDX"
```
Comment (faarwa) [inline gatsby-site/src/pages/index.js]: data.hub.name

File: gatsby-site/src/pages/index.js [hunk 4]
```diff
@@ -16,32 +16,33 @@ const MainContent = styled.div`
...
+          content={
+            "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.\n\n\n\nLorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."
```
Comment (faarwa) [inline gatsby-site/src/pages/index.js]: Is there a hub field for content yet?

File: gatsby-site/src/pages/index.js [hunk 5]
```diff
@@ -32,12 +32,7 @@ const IndexPage = ({ data }) => {
         }
       />
       <MainContent>
-        <About
-          hubName="PDX"
-          content={
-            "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.\n\n\n\nLorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."
-          }
-        />
+        <About hubName={data.hub.name} content={data.hub.about} />
```
Comment (faarwa) [inline gatsby-site/src/pages/index.js:35]: Haha ignore my previous comment

**Review by faarwa — APPROVED** [review]
A couple comments otherwise lgtm!

File: gatsby-site/gatsby-node.js (hunk 1 above)
Comment (lukewestby) [reply gatsby-site/gatsby-node.js:44]: We do not, it seems that airtable only does this with the array fields

File: gatsby-site/gatsby-node.js (hunk 2 above)
Comment (lukewestby) [reply gatsby-site/gatsby-node.js:185]: It is

**Pull request merged**